padinfo/padinfo.py

```diff
@@ -1409,3 +1410,21 @@ async def expcurve(self, ctx, start, end: Optional[int], *, query=''):
         state = ClosableEmbedViewState(original_author_id, ClosableEmbedMenu.MENU_TYPE, query,
                                        query_settings, ExperienceCurveView.VIEW_TYPE, props)
         await menu.create(ctx, state)
+
+    @commands.command()
+    async def board(self, ctx, *, message):
```

Comment:
sorry one small change that's also my fault - can you rename this function to `boardlink` ? Aradia reminded me that we have another command called `boards` and `board` autocorrects to it (you don't have the cog it's in loaded so it hasn't been a problem for you) whoops

You can leave this comment open & do it at the very end if you're used to typing ^board for testing so you don't have to change